board/safety/safety_nissan.h

```diff
@@ -182,8 +190,14 @@ static int nissan_fwd_hook(int bus_num, CAN_FIFOMailBox_TypeDef *to_fwd) {
   return bus_fwd;
 }
 
+static void nissan_init(int16_t param) {
+  controls_allowed = 0;
+  nissan_eps_location = param;
```

Comment:
it should be something like this, otherwise you can pass in any bus
```suggestion
  nissan_eps_location = param ? 1 : 0;
```